internal/table/scanner/result.go

```diff
@@ -20,9 +20,8 @@ var errAlreadyClosed = xerrors.Wrap(errors.New("result closed early"))
 type baseResult struct {
 	valueScanner
 
-	nextResultSetCounter atomic.Uint64
-	statsMtx             xsync.RWMutex
-	stats                *Ydb_TableStats.QueryStats
+	statsMtx xsync.RWMutex
+	stats    *Ydb_TableStats.QueryStats
```

Comment:
I changed this lines because structcheck linter has message:
```
internal/table/scanner/result.go:23:2: `nextResultSetCounter` is unused (structcheck)
        nextResultSetCounter atomic.Uint64
        ^
```